### DISCUSSION TITLE
Welcome to pinia-orm Discussions!

### DISCUSSION THREAD

**Discussion by CodeDredd** [CodeDredd/pinia-orm#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CodeDredd** [comment]
Hey Sven,

 

Erstmal danke für die Unterstützung. Ich denke mal, dass ich auch in Deutsch schreiben kann 😉. Bin selber aus der Umgebung von München.

Lass gerne mal zusammen quatschen. Ja ich habe Pläne für das plugin. Habe die auch mal in die Diskussion grob mal gepostet. 

Meine Aktivität ist ein wenig semi. Mit Kindern hat man halt mal Zeit, mal nicht ^^. Vor allem wenn man noch ein open source projekt hat merkt man wie das anstrengender wird ^^.

 

Würde mich auf jeden Fall mega freuen mit dir darüber zu quatschen!

Kannst mich auch in discord oder twitter erreichen. Ansonsten hätte ich auch noch skype zum Angebot.

 

Viele Grüße

Gregor

 

Von: svenhue ***@***.***> 
Gesendet: Montag, 11. April 2022 09:14
An: CodeDredd/pinia-orm ***@***.***>
Cc: Gregor Becker ***@***.***>; Author ***@***.***>
Betreff: Re: [CodeDredd/pinia-orm] Welcome to pinia-orm Discussions! (Discussion #1)

 

Hey,
my name is Sven Hüning, im from Germany and currently im finishing my dual bachelor study program and during my master i want to develop apps for midsize companies and some startups.

So basically, there is a wide range of applications i want to use it -also some realtime apps (for example digital twins for production, cnc machines... with opcua)

In an typically mid-size project i use quasar as the frontend (mostly ssr and pwa ) and django as the backend.

So... i rly like pinia and dislike vuex ;) with pinia i stay more productive.

So im looking forward to evolve this project... but is it still evolving?!

Greetings

**Reply by svenhue** [reply]
Moin, ja gerne über Discord - gibts dafür hier ein Link oder so ? :)

**Comment by CodeDredd** [comment]
> Moin, ja gerne über Discord - gibts dafür hier ein Link oder so ? :)

Also es gibt noch nicht direkt nen discord für pinia-orm. Ich glaube dafür ist das noch zu früh ^^. Kannst mich aber hierdurch adden:
CodeDredd#7204
